src/gui/spaces/spacesmodel.cpp

```diff
@@ -15,13 +15,16 @@
 
 #include "common/utility.h"
 // TODO: move models out from core
+#include "gui/guiutility.h"
 #include "gui/models/models.h"
 #include "networkjobs.h"
 
+#include <QIcon>
 #include <QPixmap>
 
 namespace {
 constexpr QSize ImageSizeC(128, 128);
+constexpr QSize ImageMarginC(ImageSizeC * 0.1);
```

Comment:
Even though the image size is static, I'd rather not use a percentage. This is a float calculation that needs to be rounded. I'd rather use a static offset.